src/managment/Contextes.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useReducer, useEffect } from 'react';
```

Comment:
I got the idea you had when you named this file - to show there are a few context providers. But it sounds a bit weird. Anyway they are used for one purpose, so it is better to rename the file, for example `GlobalStateProvider`